presentation/src/main/java/com/moez/QKSMS/feature/settings/about/AboutController.kt

```diff
@@ -49,11 +51,31 @@ class AboutController : QkController<AboutView, Unit, AboutPresenter>(), AboutVi
         showBackButton(true)
     }
 
-    override fun preferenceClicks(): Observable<PreferenceView> = (0 until preferences.childCount)
-            .map { index -> preferences.getChildAt(index) }
-            .mapNotNull { view -> view as? PreferenceView }
+    override fun preferenceClicks(): Observable<PreferenceView> {
+        fun findPreferenceViews(viewGroup: ViewGroup): List<PreferenceView> {
+            val preferences = mutableListOf<PreferenceView>()
+            for (i in 0 until viewGroup.childCount) {
+                val child = viewGroup.getChildAt(i)
+                if (child is PreferenceView) {
+                    preferences.add(child)
```

Comment:
Why did you make this change?

Reply:
Because I changed the layout and added a sub layout so I need to recursively go over all of them in order to get all of them to work.